app/javascript/articles/components/ArticleCoverImage.jsx

```diff
@@ -7,11 +7,9 @@ export const ArticleCoverImage = ({ article }) => {
       href={article.path}
       className="crayons-story__cover"
       title={article.title}
+      style={{ backgroundImage: `url(${article.main_image})` }}
     >
-      <div
-        style={{ backgroundImage: `url(${article.main_image})` }}
-        className="crayons-story__cover__image"
-      />
+      <span class="hidden">{article.title}</span>
```

Comment:
Logged in view of the featured post. Hidden text so that screen readers are all good.

Reply:
Newbie question: isn't that what `title` above does?

Reply:
See https://github.com/forem/forem/pull/10005#issuecomment-680852807